components/dialog/overview.md

```diff
@@ -45,38 +47,108 @@ The following example demonstrates how to set up the Dialog with its default con
 }
 ````
 
->caption The result from the above code snippet.
+## Predefined Dialogs
+
+Predefined Dialogs are styled substitutes to the standard confirm, alert and prompt dialogs. They are separate from the main Dialog component. [Read more about the Blazor Predefined Dialogs]({%slug dialog-predefined%}).
+
+## Header
+
+The Dialog provides header customization. [Read more about the Dialog Header]({%slug dialog-header%}).
```

Comment:
Briefly mention something valuable here, for example the developer can customize the title and toggle the close button.

```suggestion
The Dialog provides header customization. [Read more about the Dialog Header]({%slug dialog-header%}).
```